vector/src/main/res/layout/vector_settings_round_avatar.xml

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
-    android:layout_width="40dp"
-    android:layout_height="40dp">
+    android:layout_width="24dp"
```

Comment:
is this layout used anywhere else? this change might effect another screen~

Reply:
It was an error, I've revert value to those of develop branch. 
Thanks